Mp5 (#156)
OpenLiberty/certifications: microprofile/5.0/restclient/3.0/22.0.0.1-beta-TCKResults.adoc

:page-layout: certification
= TCK Results

> As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Interoperable JWT RBAC 2.0.

Reviewer: ```suggestion
As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Rest Client 3.0.
```

> == Open Liberty 22.0.0.1-beta - MicroProfile Interoperable JWT RBAC 2.0 Certification Summary

Reviewer: ```suggestion
== Open Liberty 22.0.0.1-beta - MicroProfile Rest Client 3.0 Certification Summary
```

* Product Name, Version and download URL (if applicable):
+
https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1-beta/openliberty-runtime-22.0.0.1-beta.zip[Open Liberty 22.0.0.1-beta]
* Specification Name, Version and download URL:
+

> link:https://download.eclipse.org/microprofile/microprofile-rest-client-2.0/microprofile-rest-client-spec-2.0.html[MicroProfile Rest Client 2.0]

Reviewer: Kinda the opposite problem here. I think we are using 3.0-RC3 but there is a spec for RC5 so not sure what we should have used??
```suggestion
link:https://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC3/microprofile-rest-client-spec-3.0-RC3.html[MicroProfile Rest Client 3.0-RC3]
```

Reviewer 2: can you double check whether we use 3.0-RC3? Andy McCright told me that he checked in the latest version of Rest Client.
